.github/workflows/flatpak-build.yml

```diff
@@ -46,12 +46,16 @@ jobs:
         run: echo "//registry.npmjs.org/:_authToken=${{ secrets.NPM_TOKEN }}" > ~/.npmrc
       - name: install yarn
         run: npm install --global yarn
+      - name: setup env file
+        run: echo "VITE_LD_ENVIRONMENT_ID=${{ secrets.VITE_LD_ENVIRONMENT_ID_TEST }}" > .env
       - name: Install modules.
         run: yarn setup
       - name: Build artifacts.
         run: yarn dist:linux:ci:flatpak
       - name: Prepare Flatpak
         run: yarn flatpak:prepare
+      - name: setup env file
+        run: echo "VITE_LD_ENVIRONMENT_ID=${{ secrets.VITE_LD_ENVIRONMENT_ID_TEST }}" > .env
```

Comment:
is this needed twice in the same workflow?

Reply:
no, what a mistake of copy/paste. Thanks for the catch